tests/test_chunkteacher.py

```diff
@@ -66,14 +65,6 @@ def test_normal_batchsort(self):
             task='integration_tests:chunky', batchsize=2, dynamic_batching='batchsort'
         )
 
-    @testing_utils.skipUnlessGPU
-    def test_mp_normal_bs1(self):
```

Comment:
why is this removed?

Reply:
do you think the remaining test would catch sth like `distributed_world_size > 1 ` while `batchsize = 1` (is this sth that is necessary to test?)

Reply:
No, I don't think they will.  I can bring that one back if you'd like.